src/i18n/locales/hi/common.json

```diff
@@ -0,0 +1,3 @@
+{
+  "password": ""
```

Comment:
let's copy the english version to make sure no text is missing but in the worst case (when a translation is missing we show the english text)

Reply:
Good idea, done!